app/src/test/java/mozilla/lockbox/adapter/ItemListAdapterTest.kt

```diff
@@ -28,21 +28,22 @@ class ItemListAdapterTest {
     private lateinit var parent: RecyclerView
 
     private val list = listOf<ItemViewModel>(
-            ItemViewModel("mozilla.org", "example@example.com", ""),
-            ItemViewModel("cats.org", "cats@cats.com", ""),
-            ItemViewModel("dogs.org", "woof@woof.com", "")
+        ItemViewModel("mozilla.org", "example@example.com", ""),
+        ItemViewModel("cats.org", "cats@cats.com", ""),
+        ItemViewModel("dogs.org", "woof@woof.com", "")
     )
 
     @Before
     fun setUp() {
         context = RuntimeEnvironment.application
         parent = RecyclerView(context)
         parent.layoutManager = LinearLayoutManager(context)
-        subject.updateItems(list)
+//        subject.updateItems(list)
```

Comment:
Once you're sure this call really does belong in the `@ Test`, be sure to remove this line.

Reply:
Thanks, I forgot about that, I'll add the changes